assets/js/googlesitekit/data/create-notifications-store.test.js

```diff
@@ -168,10 +168,13 @@ describe( 'createNotificationsStore store', () => {
 
 		describe( 'fetchGetNotifications', () => {
 			it( 'does not require any params', () => {
-				muteFetch( /^\/google-site-kit\/v1\/core\/site\/data\/notifications/, [] );
```

Comment:
This should continue to use `muteFetch` - a new utility added as part of that PR